### DISCUSSION TITLE
Please help me get_member() error.

### DISCUSSION THREAD

**Discussion by ins3821** [Rapptz/discord.py#6285] [General] [answered]

**Comment by mesub7** [comment]
Could you provide more information please?

**Reply by ins3821** [reply]
I solved the problem. Thank you for your interest!!

**Comment by 15696** [comment, marked as answer]
Do you have the members intent enabled? `Guild.get_member` will return None since the member is not cached because of the members intent. To fix that, you should do something like this:

```py
intents = discord.Intents.default() # turn on the non-privileged intents
intents.members = True # subscribe to the members intent
bot = commands.Bot(..., intents=intents) # pass the intents to the bot constructor
```
